internal/condition/eval/eval.go

```diff
@@ -2,13 +2,29 @@ package eval
 
 import (
 	"fmt"
+	"time"
 
 	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/openfga/openfga/internal/build"
 	"github.com/openfga/openfga/internal/condition"
+	"github.com/openfga/openfga/internal/server/config"
+	"github.com/openfga/openfga/internal/utils"
 	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+var conditionEvaluationCostHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+	Namespace:                       build.ProjectName,
+	Name:                            "condition_evaluation_cost",
+	Help:                            "A histogram of the CEL evaluation cost of a Condition in a Relationship Tuple",
+	Buckets:                         utils.LinearBuckets(1, config.DefaultMaxConditionEvaluationCost, 10),
```

Comment:
Will it be an interesting point for 0 CEL?

Reply:
I assumed that the expression `true` had cost 1. I just tested out some expressions and here's what I found.

```
condition cond1() {
  true
}
```
evaluates to a cost of 0.

```
condition cond2() {
  1 < 2
}
```
evaluates to a cost of 1.

```
condition cond3(x: int) {
  x < 2
}
```
with `x: 1` evaluates to a cost of 2.

etc..

So my assumption was wrong. I'll set the lower bound here to 0 to account for the first case `cond1`.